src/utils/embeddableUrl.tsx

```diff
@@ -0,0 +1,121 @@
+import {
+  FACEBOOK_EMBED_URL,
+  TWITCH_EMBED_URL,
+  TWITCH_SHORT_URL,
+  VIMEO_EMBED_URL,
+  VIMEO_SHORT_EVENT_URL,
+  YOUTUBE_EMBED_URL,
+  YOUTUBE_SHORT_URL_STRING,
+} from "settings";
+
+const setParamsIteratively = (urlObject: URL, urlParams?: string[]) => {
```

Comment:
`setParamsIteratively` -> `withParameters` kind of like `withAutoPlay`

Reply:
done 741d64c